<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HostSwitcherClass</class>
 <widget class="QWidget" name="HostSwitcherClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Host Switcher</string>
  </property>
  <property name="windowIcon">
   <iconset resource="host_switcher.qrc">
    <normaloff>:/images/switch.png</normaloff>:/images/switch.png</iconset>
  </property>
  <widget class="QPlainTextEdit" name="contentEditor">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>561</width>
     <height>471</height>
    </rect>
   </property>
   <property name="lineWrapMode">
    <enum>QPlainTextEdit::NoWrap</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="addItemButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteItemButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>490</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveInfoButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>490</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="itemListTableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>471</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::DoubleClicked|QAbstractItemView::SelectedClicked</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::NoPen</enum>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>1</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>200</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>200</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <column/>
  </widget>
  <widget class="QPushButton" name="loadConfigButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>490</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="helpButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>490</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="QPushButton" name="moveItemUpButton">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>180</y>
     <width>30</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>▲</string>
   </property>
  </widget>
  <widget class="QPushButton" name="moveItemDownButton">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>270</y>
     <width>30</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>▼</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="host_switcher.qrc"/>
 </resources>
 <connections/>
</ui>
